documentation/tutorials/prerequisites.md

```diff
@@ -31,3 +31,81 @@ $(npm bin)/truffle test --network test
 ```
 
 These tests will take a while to finish, but if set up correctly, all tests should pass.
+
+## Keys and Networks
+
+When using UMA infrastructure, you often have to open the `truffle console` or run a script using `truffle exec`. When
+using truffle in the context of this repository, it's important to understand what types of keys you can use and what
+to specify for the `--network` argument.
+
+### Public Networks
+
+Public networks include the Ethereum mainnet and any public testnets, like Rinkeby, Kovan, or Ropsten. If you are using
+these networks, you'll generally need to know:
+
+1. Which public network you intend to use.
+
+2. What private key you want to use and how your private keys are stored.
```

Comment:
Can we similarly make an exhaustive list of all the key "types" we support? Maybe this should be a bulleted list with a brief description?

Reply:
Are these not the keys described in the sections below, "mnemonic" and "hardware wallet"? cc @mrice32

Reply:
Correct. We have others we can list, but I think we should just start with those two.

Reply:
Matamask doesn't really work fully, and while I think you could technically hack what we have to use keys that are stored on the node, I'll like to actually test that it could work on a public network before putting it in the docs.

Reply:
Great, I'll add a sentence to the end of bullet point 2 that says, "Below is information regarding the two types of keys that are supported, mnemonics/seed phrases and hardware wallets."